src/test/java/com/mongodb/jdbc/MongoResultSetTest.java

```diff
@@ -1162,7 +1163,7 @@ void sameMetaDataOnDifferentRowsEvenWithDifferentBsonTypes() throws SQLException
                             return doc;
                         });
 
-        mockResultSet = new MongoResultSet(mongoStatement, cursor, sameMetadataSchema);
+        mockResultSet = new MongoResultSet(mongoStatement, cursor, sameMetadataSchema, false);
```

Comment:
Should we add some tests for when `isExt` is `true`?

Reply:
I was thinking I should add a test case too but I was having a hard time figuring out how to go about that :(

Is there a particular test that I can reference as a template?

Reply:
I added a test in MongoDriverTest.java based on Oliver's comment :)